### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by zixuan75**

### CODE REVIEW

**Comment by zixuan75** [comment]
Hello?

**Comment by alax99** [comment]
Well good.

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Do you think you could fix a issue?
+## Why?
+A clear and concise description about why you could fix a issue.
```
Comment (alax99) [inline PULL_REQUEST_TEMPLATE.md]: Very Good

Comment (zixuan75) [reply PULL_REQUEST_TEMPLATE.md]: Ah ok.

Comment (alax99) [inline PULL_REQUEST_TEMPLATE.md]: Hey

**Comment by zixuan75** [comment]
:ship: it!

Yes, you've got it!

**Pull request merged**